packages/examples/packages/preinstalled/src/index.tsx

```diff
@@ -64,6 +64,23 @@ export const onRpcRequest: OnRpcRequestHandler = async ({ request }) => {
       });
     }
 
+    case 'startTrace':
```

Comment:
Not for this PR, but we should add the metrics method to this Snap too so we can test it all